indico/modules/events/editing/schemas.py

```diff
@@ -39,7 +39,9 @@ class Meta:
 class EditingTagSchema(mm.ModelSchema):
     class Meta:
         model = EditingTag
-        fields = ('id', 'name', 'color', 'system')
+        fields = ('id', 'code', 'title', 'color', 'system', 'verbose_title')
+
+    verbose_title = fields.Function(lambda tag: '{}: {}'.format(tag.code, tag.title))
```

Comment:
hm, property on the model vs this? if we need it in server-side code (notification emails?) at some point the property would be better

Reply:
Good point.